tasks/testwasher.py

```diff
@@ -1,6 +1,6 @@
 import json
 from collections import defaultdict
-from typing import List
+from typing import List, Tuple
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏You can remove this import, use `from __future__ import annotations` and then replace `List` -> `list` and `Tuple` -> `tuple`